lib/transaction/index.js

```diff
@@ -414,8 +502,28 @@ transaction.prototype.requestStrategyAuth = function requestStrategyAuth(strateg
       self.eventSequencer.emit('error', error);
     });
 
+    self.authVerificationStep = verificationStep;
+
     callback(null, verificationStep);
   });
 };
 
+transaction.prototype.getAuthVerficationStep = function getAuthVerficationStep() {
+  if (!self.authVerificationStep) {
+    throw new errors.InvalidStateError('cannot get enrollment confirmation ' +
+    'step, you must call .requestAuth first');
+  }
+
+  return self.authVerificationStep;
+};
+
+transaction.prototype.getEnrollmentConfirmationStep = function getEnrollmentConfirmationStep() {
```

Comment:
This can be easily converted in @nikolaseu proposal of adding the method confirmEnrollment to the transaction, just call getEnrollmentConfirmationStep().confirm(...) on that method.